src/Raven.Client/Documents/Commands/Batches/CountersBatchCommandData.cs

```diff
@@ -30,6 +30,7 @@ public CountersBatchCommandData(string documentId, CounterOperation counterOpera
         public string Id { get; }
         public string Name { get; }
         public string ChangeVector { get; }
+        public bool HasIncrementOperation { get; set; }
```

Comment:
Why do we need this property when we have `HasIncrement`?

Reply:
removed this property